ext/dom/tests/DOMNode_isEqualNode.phpt

```diff
@@ -200,7 +200,8 @@ echo "--- Test documents ---\n";
 $dom1Clone = clone $dom1;
 var_dump($dom1->documentElement->isEqualNode($dom1Clone->documentElement));
 var_dump($dom1->documentElement->isEqualNode($dom2->documentElement));
-var_dump($dom1->isEqualNode($dom1Clone));
+// TODO: re-enable this once this is figured out: https://gitlab.gnome.org/GNOME/libxml2/-/issues/634
+//var_dump($dom1->isEqualNode($dom1Clone));
```

Comment:
Maybe XFAIL this test rather than comment the line? Or split this seemingly large test into multiples and XFAIL just that? Anyway I am fine with this just being commented out

Reply:
The libxml maintainer fixed this sooner than expected. The fix will be in 2.12.2. So I'll just mark this test as XFAIL for 2.12.0 & 2.12.1.